warehouse/templates/search/results.html

```diff
@@ -35,9 +35,6 @@ <h3 class="package-snippet__title">
 
```

Comment:
Looks like the line above this one (`% set freq = best_guess.freq %}`) can be removed as well.